### DISCUSSION TITLE
InnerSource Certification

### DISCUSSION THREAD

**Discussion by mishari** [InnerSourceCommons/InnerSourceMarketing#159] [Ideas]
I would like to start discussions for providing certifications as in South East Asia, certifications are highly regarded and are heavily incentivized by Human Resource policies. This is one of the factors that drove Agile adoption.

**Comment by Danese** [comment]
Mishari,

I agree that certs are super-important in Asia and would likely drive
adoption there. To offer them as an organization we would need to contract
a third party to actually issue the certs (because of liability issues in
the US). I helped create the Node.js certification process when we spun up
the Node Foundation and that was a very necessary design choice. The actual
“exam” was created by a handful of practitioners (with a grant from IBM),
and the content was reviewed by the Node community (so it was a joint
effort that helped bring the community together), but we couldn’t roll it
out until we found a firm that would grade the exams and issue the cert.

Might be a good idea to identify a firm to certify results first.

D

**Comment by mishari** [comment]
I also need to add a counterpoint: the Agile scene is in a bit of a mess with certifications being no indicator of quality nor understanding. With reasonably rapid community driven growth and adoption already taking place, perhaps the InnerSource movement can avoid going down the Agile road by not endorsing certificates and see where it takes us.
